definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+# Definition of done
+
+This document clarifies, when an product backlog item is done. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api
+* [ ] API diagrams
```

Comment:
A link to API diagrams would also be helpful  (or where to find them, see comment above)